src/plugins/es_ui_shared/static/forms/hook_form_lib/types.ts

```diff
@@ -193,6 +193,11 @@ export interface ValidationFuncArg<I extends FormData, V = unknown> {
   };
   formData: I;
   errors: readonly ValidationError[];
+  customData: {
```

Comment:
Is `customData` a required property of `ValidationFuncArg`? To me it seems like it should be an optional property, as not all forms need a validation with `customData`.

Reply:
Yes the `customData` object will always be there in the validator. That does not mean that the consumer needs to pass custom data 😊 If he doesn't, its `value` will simply be `undefined`.